NOTICE.txt

```diff
@@ -0,0 +1,40 @@
+NOTICES AND INFORMATION
+Do Not Translate or Localize
+
+This software incorporates material from third parties.
+Microsoft makes certain open source code available at https://3rdpartysource.microsoft.com,
+or you may send a check or money order for US $5.00, including the product name,
+the open source component name, platform, and version number, to:
+
+Source Code Compliance Team
+Microsoft Corporation
+One Microsoft Way
+Redmond, WA 98052
+USA
+
+Notwithstanding any other terms, you may reverse engineer this software to the extent
+required to debug changes to any libraries licensed under the GNU Lesser General Public License.
+
+---------------------------------------------------------
+
+vscode-cpptools 6.1.0 - MIT
```

Comment:
This is not a 3rd party license -- it's 1st party license, i.e. it's already covered by https://github.com/microsoft/vscode-cpptools-api/blob/main/LICENSE .

Reply:
Let me follow up with Bri, thanks.